Client/game_sa/CAutomobileSA.h

```diff
@@ -35,6 +40,13 @@ static_assert(sizeof(CBouncingPanelSAInterface) == 0x20, "Invalid size for CBoun
 class CAutomobileSAInterface : public CVehicleSAInterface
 {
 public:
+    void SetPanelDamage(std::uint8_t panelId, bool breakGlass, bool spawnFlyingComponent = true);
+
+    CObjectSAInterface* SpawnFlyingComponent(const eCarNodes& nodeId, const eCarComponentCollisionTypes& collType)
+    {
+        return ((CObjectSAInterface*(__thiscall*)(CAutomobileSAInterface*, eCarNodes, eCarComponentCollisionTypes))0x6a8580)(this, nodeId, collType);
+    }
```

Comment:
This should return a `CPhysicalSAInterface*`

Reply:
```suggestion
    CPhysicalSAInterface* SpawnFlyingComponent(const eCarNodes& nodeId, const eCarComponentCollisionTypes& collType)
    {
        return ((CPhysicalSAInterface*(__thiscall*)(CAutomobileSAInterface*, eCarNodes, eCarComponentCollisionTypes))0x6a8580)(this, nodeId, collType);
    }
```

Reply:
Why? In my IDB and on gta-reversed, it's CObject*

Reply:
Could you find an actual game function that expects CObject* as a return value?

Reply:
I should've looked into `CAutomobile::SpawnFlyingComponent` first, you're right